opentelemetry-sdk/src/opentelemetry/sdk/error_handler/__init__.py

```diff
@@ -69,7 +70,7 @@ def _handle(self, error: Exception, *args, **kwargs):
 
 class ErrorHandler(ABC):
     @abstractmethod
-    def _handle(self, error: Exception, *args, **kwargs):
+    def _handle(self, error: Exception, *args, **kwargs) -> None:  # type: ignore[misc, no-untyped-def]
```

Comment:
The `no-untyped-def` can be removed by properly type hint it:
```suggestion
    def _handle(self, error: Exception, *args: Any, **kwargs: Any) -> None:
```
Why is the `misc` there?